telluric/features.py

```diff
@@ -54,9 +55,25 @@ def serialize_properties(properties):
             # https://docs.python.org/3.4/library/json.html#json.JSONEncoder
             # so we convert to string
             new_properties[attr_name] = str(attr_value)
-
     return new_properties
 
+def from_assets(assets):
+    """ create a raster from assets, assets is a dictonary of links like described
+        in the stacs inteface https://github.com/radiantearth/stac-spec/tree/master/json-spec/examples
+
+        * currently we support a single raster asset, in the feature we could support more
+    """
+    raster = None
+    for key, val in assets.items():
+        href = val.get("href", None)
+        bands = val.get("bands", None)
+        if href:
+            raster = GeoRaster2.open(href, band_names=bands)
+            break
+    return raster
```

Comment:
I would expect this to return a list with a single raster

Reply:
I had in mind something like RasterCollection, which is a single composite raster. 

Very similar to GeometryCollection which a single composite vector, 

Therefore when we support more then one raster we still should return a single object and not a list

Reply:
changed the name